Content.Server/Shuttles/Systems/ShuttleSystem.FasterThanLight.cs

```diff
@@ -478,12 +479,28 @@ private void UpdateFTLArriving(Entity<FTLComponent, ShuttleComponent> entity)
 
         if (!Exists(entity.Comp1.TargetCoordinates.EntityId))
         {
-            // Uhh good luck
-            // Pick earliest map?
-            var maps = EntityQuery<MapComponent>().Select(o => o.MapId).ToList();
-            var map = maps.Min(o => o.GetHashCode());
+            // Fallback time.
+            // Try to find a map with a station. Otherwise just find /any/ map.
```

Comment:
One other approach would be to first check maps with an FTLDestination that would allow this shuttle to FTL to them and pick one of those at random, before doing fallbacks like this, since this could break if for example the nukie outpost was made to be a station (for some reason...). That feels pretty unlikely though.